### ISSUE TITLE
Are you supporting macOS on ARM?

### ISSUE THREAD

**Issue by timsnyder** [sparcians/map#319]
Just FYI, conda-forge is starting to build stuff for what they call `osx_arm64`.  I don't know if we're going to start having people with ARM based macbooks who need to develop stuff.

**Comment by jackjchen** [comment]
It would help if I could build locally for osx-arm64, but I am not so much needing CI for this target platform. I tried to test https://github.com/sparcians/map/blob/master/scripts/create_conda_env.sh by hacking https://github.com/sparcians/map/blob/master/.ci_support/osx_64_.yaml with a few relevant values from https://github.com/conda-forge/conda-feedstock/blob/main/.ci_support/osx_arm64_python3.8.____cpython.yaml

It seems that the conda render does not expand the wildcard specs for python in the `build` section (`host` section is fine), and then the resulting environment.yml has the 2 incompatible merge specs for the conda-env create step:
   - python=3.8.\*=\*_cpython
   - python=3.8.13=h7e095e3_0_cpython

I was able to continue command line env creation by removing 1 of the python deps (and the no-spec texlive-core)

Does it seem this is an issue with the render from conda/conda for osx-arm64?

**Comment by timsnyder-siv** [comment]
I'll take a look and see if I can reproduce the problem on my Mac.

**Comment by klingaard** [comment]
Some folks in our Taiwan office have ARM-based Macs.  Not sure if they've tried the argos tools (which is what you really need the conda env for).  @2henwei

**Comment by 2henwei** [comment]
I did try building conda environment and encounter some issues, but I wasn't using `osx_arm64`. Did some dirty works to build the conda eventually, but `argos` just didn't work well (it could launch, but had some rendering issues).

**Comment by knute-sifive** [comment]
Hey @timsnyder, is this issue good for closure?

**Comment by jackjchen** [comment]
Issue looks solved regarding my earlier comment -- the scripts/create_conda_env.sh works fine for my local osx-arm64 dev environment after #419.

If anyone has interest, a separate issue would be for Linux-aarch64. There are problems with the create script with the conda solver not able to solve the rendered deps/versions and wxpython is not even found on conda-forge for aarch64.
